xbmc/GUIInfoManager.cpp

```diff
@@ -3732,6 +3732,16 @@ const infomap container_str[]  = {{ "property",         CONTAINER_PROPERTY },
 ///                  _string_,
 ///     Name of the repository the add-on originates from.
 ///   }
+///   \table_row3{   <b>`ListItem.ExpirationDate`</b>,
+///                  \anchor ListItem_ExpirationDate
+///                  _string_,
+///     Expiration date of the selected item in a container, empty string if not supported
+///   }
+///   \table_row3{   <b>`ListItem.ExpirationTime`</b>,
+///                  \anchor ListItem_ExpirationTime
+///                  _string_,
+///     Expiration time of the selected item in a container, empty string if not supported
+///   }
```

Comment:
You need to add a **`\`** before **`,`** inside text lines! Also seen on `ListItem.EpgEventTitle` missing (may be also somewhere else).

```
Expiration time of the selected item in a container\, empty string if not supported
```

Thats a stupid bad thing on doxygen by use of my table style.